server/routerlicious/packages/routerlicious/src/deli/index.ts

```diff
@@ -34,7 +34,9 @@ export async function deliCreate(config: Provider): Promise<core.IPartitionLambd
 
     // Generate tenant manager which abstracts access to the underlying storage provider
     const authEndpoint = config.get("auth:endpoint");
-    const tenantManager = new services.TenantManager(authEndpoint);
+    const defaultHistorianUrl = config.get("worker:blobStorageUrl");
+    const defaultInternalHistorianUrl = config.get("worker:internalBlobStorageUrl") || defaultHistorianUrl;
```

Comment:
I wonder if we should add a new config value so that we can preserve existing functionality. The rest of your code here still allows for storage URL to be set by the tenant if there is no config override, which I think is great. However, the code here undoes that by always making the override value exist. My thought is that we should add something like `storage:internalHistorianUrlOverride` _if_ we do in fact want to preserve the ability for tenant metadata to set the storage url.

Also, as a note, generally try to use `??` instead of `||` because it is more specific. `||` means "if previous value is **falsey**, use this following value", but `??` means "if previous value is **undefined**, use this following value". In the case of configs, this is extra important for making sure to not override the value "false" with a default of "true".

Reply:
I agree. Also, how did we previously use that existing `worker` config, such as `worker:blobStorageUrl` and `worker:internalBlobStorageUrl`? Were those values/configs supposed to be used as in here, or is it just a coincidence that they share the same values we want, but happen to have different semantics?

Reply:
We have the worker.blobStorageUrl and worker:internalBlobStorageUrl to represent the defaultHistorianUrl and defaultInternalHistorianUrl used by the cluster. 
They are already been used here from line 93 - line 94 here:
https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts#:~:text=const%20defaultHistorianUrl%20%3D%20config,get(%22worker%3AinternalBlobStorageUrl%22)%20%7C%7C%20defaultHistorianUrl%3B


We might use the `||`, since it is quoted the same in the runnerFactory.ts. I think it might have the logic of checking the empty string for config.get("worker:internalBlobStorageUrl").